audit/projects/k8s-release/buckets/k8s-release-asia/metadata.txt

```diff
@@ -0,0 +1,70 @@
+gs://k8s-release-asia/ :
```

Comment:
This was me claiming the bucket name by manually creating the bucket for https://github.com/kubernetes/k8s.io/issues/2396